runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -659,6 +659,10 @@ private List<PTransformOverride> getOverrides(boolean streaming) {
 
       try {
         overridesBuilder.add(KafkaIO.Read.KAFKA_READ_OVERRIDE);
+        overridesBuilder.add(
+            PTransformOverride.of(
+                KafkaReadWithRedistributeOverride.matcher(),
+                new KafkaReadWithRedistributeOverride.Factory()));
```

Comment:
ideally move Override to KafkaIO as `KafkaIO.Read.KAFKA_READ_OVERRIDE` above. This keeps reference to KafkaIO for DataflowRunner minimum (ease import)